### ISSUE TITLE
Support Item description  for HUD mods like Jade/Hwyla/Waila for sub-parts

### ISSUE THREAD

**Issue by rhullah** [CyclopsMC/IntegratedTunnels#305]
#### Issue type:

- :heavy_plus_sign: Feature request 

____

#### Short description:

When using a mod like Jade to have a HUD of what you're looking at, when focused on sub parts that connect to Logic Cables the HUD only shows the Logic Cable instead of the sub-part you're looking at like interfaces, importers, exporters, terminals, etc...all of them.  I often don't remember the color scheme for each part and don't remember which one is which.@@

**Comment by rubensworks** [comment]
Thanks for the suggestion!

**Comment by rubensworks** [comment]
ID should have Waila support already. Doesn't it work for you? (if so, please report this as a bug issue)
AFAIK, Hwyla uses the same API, so that should then also work.

TOP is also already supported.

I didn't know yet about Jade. I guess that's the new kid on the block?

**Comment by rhullah** [comment]
Yeah, Jade seems to be used in more modern modpacks. I'm not sure how support for each is implemented, but I though Jade was supposed to be like a drop-in replacement of Waila. If this should be a bug instead, could you just change it's type as a bug? Otherwise, I can create a new one as a bug.

Here's what I see for Jade:
<img width="351" alt="image" src="https://github.com/user-attachments/assets/a3f9536d-0de7-4e57-a11c-928c77a0937a">
